### PR TITLE
test: scala_library with reference.conf resource

### PR DESCRIPTION
**Pull request by bjchambers**

### CODE REVIEW

File: rules/common/private/utils.bzl [hunk 1]
```diff
@@ -152,7 +152,7 @@ def action_singlejar(
     args.add_all("--sources", inputs)
     args.add_all("--resources", ["{}:{}".format(value.path, key) for key, value in resources.items()])
     args.add("--output", output)
-    args.add("--warn_duplicate_resources")
+    #args.add("--warn_duplicate_resources")
```
Comment (SrodriguezO) [inline rules/common/private/utils.bzl]: I think we should go ahead and remove this.

Comment (SrodriguezO) [inline rules/common/private/utils.bzl]: Can you remove the commented-out line?

File: tests/resources/reference_conf/BUILD [hunk 2]
```diff
@@ -0,0 +1,14 @@
+load("@rules_scala_annex//rules:scala.bzl", "scala_library")
+
+scala_library(
```
Comment (SrodriguezO) [inline tests/resources/reference_conf/BUILD]: Thanks for adding tests!

**Comment by SrodriguezO** [comment]
@bjchambers I see that you force pushed, but I don't see the changes, so I'm wondering if you forgot to amend the commit first?

**Comment by bjchambers** [comment]
> @bjchambers I see that you force pushed, but I don't see the changes, so I'm wondering if you forgot to amend the commit first?

Whoops. Maybe? Repushed.

**Pull request merged**